catalog/to-consul/resource.go

```diff
@@ -289,6 +294,11 @@ func (t *ServiceResource) generateRegistrations(key string) {
 		},
 	}
 
+	hostname, err := os.Hostname()
+	if err == nil {
```

Comment:
If there was an error, we'll need to log it and then we can set the `hostname` to `<unknown>`.